JapaneseFilter/sections/specific.txt

```diff
@@ -47,6 +47,7 @@ bizushiki.com##.side-widget-area > div[id^="text-"]
 bizushiki.com##div[class^="g g-"]
 shotworks.jp###adSqb
 tegaki.pipa.jp###Ad300x250
+iphoroid.jp##div[style="margin-right:auto; margin-left:auto; width:620px;"]
```

Comment:
Better to add to `kirari-media.net##div[id][style="height:250px; width:300px;"]`.

Reply:
I see. I will fix.